CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ enable_testing()
 add_definitions(-DLOG_USE_COLOR)
 
 option(BUILD_TESTS "True if build tests" on)
+set(OPENSSL_VERSION_NUMBER  0x11100000L)
```

Comment:
Will this cause the build to fail if the version of OpenSSL that a customer is building with doesn't match this version of OpenSSL?

Reply:
They should not fail. As we statically link and do archive all libs into libSnowflake.so the application should link with the client specific openssl.